apis/vshn/v1/dbaas_vshn_keycloak.go

```diff
@@ -120,6 +120,11 @@ type VSHNKeycloakServiceSpec struct {
 	// The themes need to be be placed in the `/themes` directory of the custom image.
 	// the providers need to be placed in the `/providers` directory of the custom image.
 	CustomizationImage VSHNKeycloakCustomizationImage `json:"customizationImage,omitempty"`
+
+	// Configuration can be used to provide a configmap containing configurations for the
+	// keycloak instance. The config is a JSON file based on the keycloak export files.
+	// The referenced configmap, must have the configuration in a field called `keycloak-config.json`
+	Configuration *string `json:"configuration,omitempty"`
```

Comment:
I assume the config cannot contain any sensitive data?

If it can, then I would rather go with a secretRef here instead of config maps.

Reply:
I'm actually not 100% sure about that. I'll have to check this with Inventage.

Reply:
I've improved the code to allow for specifying env variables via a {{secret}} that can then be referenced in the configuration in the {{configMap}}